save-cosv/src/main/kotlin/com/saveourtool/save/cosv/repository/OrganizationRepository.kt

```diff
@@ -33,4 +35,58 @@ interface OrganizationRepository {
         nativeQuery = true,
     )
     fun getOrganizationByName(@Param("name") name: String): Organization
+
+    /**
+     * @param userId
+     * @param organizationId
+     * @return found [Role]
+     */
+    @Query(
+        value = "select role from save_cloud.lnk_user_organization where user_id = :user_id and organization_id = :organization_id",
```

Comment:
why we need native quires here?

Reply:
two different data sources in one app and Spring cannot work with it well

Reply:
but may be Vlad can use this: https://www.baeldung.com/spring-boot-configure-multiple-datasources

Reply:
It's like that. In the next few PR I'll try to do it a differently